src/store/actions.js

```diff
@@ -38,6 +38,30 @@ export default {
       .filter((transaction) => !transactions.pending.find((tx) => tx?.hash === transaction?.hash))
       .map((transaction) => ({ ...transaction, pending: true }));
   },
+  // TODO: remove it after https://github.com/aeternity/tipping-community-backend/issues/405, 406 will be resolved
```

Comment:
why remove it, seems like we will need it regardless?

Reply:
I've rephrased it.